### PR TITLE
docs(builder): add ci scripts

### PR DESCRIPTION
**Pull request by johannesschobel**
added my ci scripts (matrix and individual) to the readme file (see #599 )

### CODE REVIEW

File: libs/builders/src/semantic-release/README.md [hunk 1]
```diff
@@ -208,6 +208,98 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }} # Personal access token with repo permissions
 ```
 
+You can also use a `Matrix Setup` for your projects to be released. This works as follows:
```
Comment (samuelfernandez) [inline libs/builders/src/semantic-release/README.md:211]: **issue** Friend, though this works, it is not something I'd like to promote. The reason is that it relies on hardcoding dependencies, something that the builder is actually doing (when it works well).

**suggestion** Would you mind removing the matrix example? Thanks!

File: libs/builders/src/semantic-release/README.md [hunk 2]
```diff
@@ -208,6 +208,98 @@ jobs:
...
+      - name: Fetch latest base branch
+        run: git fetch origin main
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Setup NPM
+        uses: ng-easy/npm-setup@v2
```
Comment (samuelfernandez) [inline libs/builders/src/semantic-release/README.md:290]: **nitpick** We can add the `nx-key` param so as to benefit from local cache of the build process

```suggestion
        uses: ng-easy/npm-setup@v2
        with:
          nx-key: build
```

File: libs/builders/src/semantic-release/README.md [hunk 3]
```diff
@@ -208,6 +208,98 @@ jobs:
...
+
+      - name: Build Package
+        run: npx nx run ${{ github.event.inputs.project }}:build
```
Comment (samuelfernandez) [inline libs/builders/src/semantic-release/README.md:293]: **question** could you make it work with ` mansagroup/nrwl-nx-action@v2`? In theory you could use the same interpolation approach when passing it to an input

**Review by samuelfernandez — CHANGES_REQUESTED** [review]
@johannesschobel some comments on my side, thanks for the contrib!

**Comment by johannesschobel** [comment]
i will take a look at it, of course!

File: libs/builders/src/semantic-release/README.md (hunk 2 above)
Comment (johannesschobel) [reply libs/builders/src/semantic-release/README.md:290]: of course i will add this

File: libs/builders/src/semantic-release/README.md [hunk 4]
```diff
@@ -208,6 +208,71 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }} # Personal access token with repo permissions
 ```
 
+You can also use the `Inputs` feature provided by `GitHub` to create a user-interface that lets you select a specific project to be released.
```
Comment (samuelfernandez) [inline libs/builders/src/semantic-release/README.md:211]: One additional comment I've realized about. This approach can cause a problem if projects are dependent. For example, imagine that there are changes to `project-a` and `project-b`, `project-b` depends on `project-a`, and you only release `project-b`. The released package will have the dependency on the old version, and might break. So we should clarify that:

```suggestion
If the packages don't depend on each other, you can also use the `Inputs` feature provided by `GitHub` to create a user-interface that lets you select a specific project to be released. If there were dependent, previous approach will take care of the dependencies releasing packages in order.
```

**Review by samuelfernandez — CHANGES_REQUESTED** [review]
Sorry, one last comment, thanks for the patience!

**Comment by johannesschobel** [comment]
Dear @samuelfernandez ,
thanks for the suggestion - i have added the additional information.

**Review by samuelfernandez — APPROVED** [review]
Great! Let's merge this one

**Pull request merged**